AccountService.js

```diff
@@ -40,4 +40,72 @@ export class AccountService {
     });
     return response.data;
   }
+
+  /**
+   * @method accounts
+   * @param {Object} config
+   * @param {string} config.url the base url
+   * @param {string} config.email the user's email
+   * @param {string} config.after an account's DID
+   * @param {number} config.limit how many accounts to return
+   * @return {Array} data
+   * @description calls on the adminr route and returns all
+   * accounts that match the email passed in.
+   * the cursor should be an account's DID
+   * and the limit is number.
+   */
+  async accounts({url = this.config.urls.base, email, after, limit}) {
+    const response = await axios.get(url, {
+      params: {email, after, limit},
+      headers: {'Accept': 'application/ld+json, application/json'}
+    });
+    return response.data;
+  }
+
+  /**
+   * @method update
+   * @param {Object} config
+   * @param {string} config.url
+   * @param {string} config.id
+   * @param {number} config.sequence
+   * @param {Array<Object>} config.patch
+   * @return {Void}
+   * @description id is an account's DID,
+   * sequence is an integer
```

Comment:
I think you can just put {integer} above in the type.  It is my understanding that one can specify anything they want in those type declarations.  Then this line can be eliminated.

Reply:
in this case (unlike with the json validator) integer is not a type js-doc actually supports: http://usejsdoc.org/tags-type.html

Although I am 100% sure there is an extension that does.
just so you know it does have to valid otherwise when you run the actual js-doc node app the parser will fail in other words if we actually work out a good documentation system we can link between projects then in the html, yml, or whatever format we decide to export our docs to the docs will produce real hyper links to each other. so for instance if you use @param {express.Request} req when we build the docs it will look for express's docs for Request when you click on Request you will actually be sent to express' doc's for Request. Simlarly any classes we build with docs if used as a param will show up as links to their docs.

We might also consider using name spaces or modules:

http://usejsdoc.org/tags-namespace.html

http://usejsdoc.org/tags-module.html

I think each project exports a module so we can use a module and then have a name space bedrock-account for instance would be a namespace with modules:
bedrock-web-account
bedrock-account
bedrock-account-http

p.s. we should really just install js-doc and run it right now before we make any large parser errors.

Reply:
Additionally sites like: https://readthedocs.org/

will automatically compile your docs and create a nice pleasant little site out of your docs.

https://www.npmjs.com/package/jsdoc-sphinx

and other projects will turn js-doc into the rst format read the docs likes.

read the docs example:

https://bootstrap-datepicker.readthedocs.io/en/latest/

Reply:
This is the source of my information about this.  Agree to another comment, it would be nice to know if any of the jsdoc stuff we're doing is valid! :)

Reply:
And here's the link.  https://stackoverflow.com/a/14131405

Reply:
let me install js-doc in this project and see what happens.

Reply:
Doc linting could help with this: https://github.com/gajus/eslint-plugin-jsdoc

Reply:
@davidlehn why not add that to this project right now and see how it goes?